packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1605,7 +1722,15 @@ export default class TradeStore extends BaseStore {
         return findFirstOpenMarket(active_symbols, markets_to_search);
     }
 
-    setBarrierChoices(barrier_choices) {
+    setStrikeChoices(strike_prices: string[]) {
```

Comment:
@henry-deriv `setStrikeChoices` need to be removed. we are using `setBarrierChoices` instead after Turbos PR https://github.com/binary-com/deriv-app/pull/7514/files#diff-ac84bdaae0a85981966f330c025656c3aaa66895a9376e738bb503d088594ec0 was released.

Reply:
removed